index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>John Buhendwa</title>
+</head>
+<body>
+    <header>
+        <div id="logo">Welcome</div>
+        <nav><i id="menu" class="fa fa-bars"></i></nav>
+    </header>
+    <main>
+        <section class="headline-section">
+            <h1>Hey there. I'm John Buhendwa <span>I'm a software developer</span></h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            <div class="social-media">
+                <i class="fa fa-twitter"></i>
+                <i class="fa fa-linkedin"></i>
+                <i class="fa fa-medium"></i>
+                <i class="fa fa-github"></i>
+                <i class="fa fa-angellist"></i>
+            </div>
```

Comment:
- [ ] Good job using creating the list :100: :+1: , but as a convention, you should prefer using  `<ul>` instead of `<div>` and wrapping each `<i>` element with `<li>` tag. Also, you should use `<a>` to link each `<i>` element to the respective social media in the future. [Here](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md) you can find more info about HTML semantics.